lib/TeleIrc.js

```diff
@@ -497,6 +470,10 @@ class TeleIrc {
       message
     );
   }
+
+  status(message) {
```

Comment:
Is this a debugging relic?

Reply:
It is not... typically when I have a class that reports logging, I have a single function called "Status" that handles how the logging takes place.  Maybe a timestamp wants to come before the message.  Maybe we want to do Console.error instead of Console.log.

I haven't converted all of the other console.log calls with this pull; I was planning on doing that when I attempt to make the Telegram side less of a cluster :P.